nemo/collections/asr/models/clustering_diarizer.py

```diff
@@ -258,9 +259,14 @@ def _run_vad(self, manifest_file):
             shift_length_in_sec=self._vad_shift_length_in_sec,
             num_workers=self._cfg.num_workers,
         )
-        AUDIO_VAD_RTTM_MAP = deepcopy(self.AUDIO_RTTM_MAP.copy())
-        for key in AUDIO_VAD_RTTM_MAP:
-            AUDIO_VAD_RTTM_MAP[key]['rttm_filepath'] = os.path.join(table_out_dir, key + ".txt")
+
+        AUDIO_VAD_RTTM_MAP = {}
+        for key in self.AUDIO_RTTM_MAP:
+            if os.path.exists(os.path.join(table_out_dir, key + ".txt")):
+                AUDIO_VAD_RTTM_MAP[key] = deepcopy(self.AUDIO_RTTM_MAP[key])
+                AUDIO_VAD_RTTM_MAP[key]['rttm_filepath'] = os.path.join(table_out_dir, key + ".txt")
+            else:
+                logging.warning(f"no .frame file found for {key} due to zero or negative duration")
```

Comment:
.frame here?

Reply:
We create .frame then .txt vad right? 
I can change it to something like no VAD file found otherwise to make it general

Reply:
yes please. we typically use .median but .frame/.median/.mean are all the options. More general would be great!